lua/lspconfig/lelwel_ls.lua

```diff
@@ -0,0 +1,25 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+configs.lelwel_ls = {
+  default_config = {
+    cmd = { 'lelwel-ls' },
+    filetypes = { 'llw' },
+    root_dir = util.path.dirname,
```

Comment:
I would use `.git` as the root dir, see the utility function for finding the git ancestor

Reply:
Done.